crates/ra_ide/src/syntax_highlighting.rs

```diff
@@ -398,6 +398,18 @@ fn highlight_element(
                 _ => h,
             }
         }
+        PREFIX_EXPR => {
+            let prefix_expr = element.into_node().and_then(ast::PrefixExpr::cast)?;
+            let expr = prefix_expr.expr()?;
+            let ty = sema.type_of_expr(&expr)?;
+            if !ty.is_raw_ptr() {
+                return None;
+            }
+
+            let mut h = Highlight::new(HighlightTag::Function);
```

Comment:
I don't know much about the highlighting code. Is `Function` the right tag to use here?

Also, shouldn't this check that this is a deref operation?

Reply:
Oh, I forgot to follow-up on this. It seems I can't just add the unsafe modifier without some existing highlight tag. I used function initially just to see if I could get the ui tests working and forgot to circle back. Thanks for pointing this out. Who would be the best to bring in for this one? There's no tag for expressions, so I wasn't sure whether to add a new one or if one of the existing ones was meant to be used in cases like this.

Reply:
Adding the case for the deref token. Still not sure who to contact about which tag to use or whether to create a new one.

Reply:
Added `HighlightTag::Operator`.